core/src/main/java/apoc/convert/Json.java

```diff
@@ -231,6 +231,73 @@ public Stream<MapResult> toTree(
                 .map(MapResult::new);
     }
 
+    @Procedure("apoc.paths.toJsonTree")
+    @Description(
+            "apoc.paths.toJsonTree([paths],[lowerCaseRels=true], [config]) creates a stream of nested documents representing the at least one root of these paths")
```

Comment:
Would be good with different a different description for the new toTree so that it is possible to understand what the difference is.